src/dao/PourChartDAO.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import type { PourChartParams, PourChartDataItem } from '../index';
+import type DAOResult from './DAOResult';
+
+import DAO from './DAO';
+import { DAO_ENTITIES } from '../constants';
+import DefaultTranslator from '../translators/DefaultTranslator';
+
+class PourChartDAO extends DAO<PourChartDataItem, PourChartDataItem> {
```

Comment:
Ahh... the outer one should be the dataset.  The inner one could be `PourChartDataEntry`